apps/rush-lib/src/utilities/Utilities.ts

```diff
@@ -759,63 +766,25 @@ export class Utilities {
     command: string,
     args: string[],
     workingDirectory: string,
-    stdio:
-      | 'pipe'
-      | 'ignore'
-      | 'inherit'
-      | (number | 'pipe' | 'ignore' | 'inherit' | 'ipc' | stream.Stream | null | undefined)[]
-      | undefined,
+    stdio: IExecutableSpawnSyncOptions['stdio'],
     environment?: IEnvironment,
     keepEnvironment: boolean = false
-  ): child_process.SpawnSyncReturns<Buffer> {
-    const options: child_process.SpawnSyncOptions = {
-      cwd: workingDirectory,
-      shell: true,
+  ): child_process.SpawnSyncReturns<string> {
+    const options: IExecutableSpawnSyncOptions = {
+      currentWorkingDirectory: workingDirectory,
       stdio: stdio,
-      env: keepEnvironment
+      environment: keepEnvironment
         ? environment
         : Utilities._createEnvironmentForRushCommand({ initialEnvironment: environment })
     };
 
-    // This is needed since we specify shell=true below.
```

Comment:
Ensure removing this doesn't break lifecycle scripts and global commands.

Reply:
Reverted this part of the change to be scoped to a different PR. It's still something we should do, since running commands in shell mode is bad practice, but will be dealt with separately.